src/main/java/mate/academy/springboot/datajpa/dto/CategoryRequest.java

```diff
@@ -0,0 +1,13 @@
+package mate.academy.springboot.datajpa.dto;
+
+import javax.validation.constraints.NotBlank;
+import lombok.Data;
+import lombok.experimental.Accessors;
+
+@Data
+@Accessors(chain = true)
+public class CategoryRequest {
```

Comment:
```suggestion
public class CategoryRequestDto {
```
the same for other dtos